environment.yml

```diff
@@ -1,4 +1,6 @@
-name: nf-core-epitopeprediction-1.0.0
+# You can use this file to create a conda environment for this pipeline:
+#   conda env create -f environment.yml
+name: nf-core-epitopeprediction-1.0.0dev
```

Comment:
Version should be higher than 1.0.0 now, no?
1.1.0.dev or 1.0.1dev

This would apply to several lines in the PR

Reply:
good point, I've updated this now